infinispan-embedded/src/test/resources/infinispan-dist-noglobal.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Hibernate OGM, Domain model persistence for NoSQL datastores
+ ~
+ ~ License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ ~ See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+  -->
+
+<!--
+    This is a testing configuration, running in clustered mode.
```

Comment:
Could you add a line to explain what' s different in this configuration and why? In a couple of weeks we won't remember that this configuration file is used to test the missing property